apps/protocol-frontend/src/contexts/ContributionContext.tsx

```diff
@@ -250,7 +200,7 @@ export const ContributionsContextProvider: React.FC<
           isClosable: true,
           position: 'top-right',
         });
-        await getUserContributions();
+        queryClient.invalidateQueries(['contributionList']);
```

Comment:
Once this PR is merged in we'll have to rectify this with the create Contribution mutation since this functionality is moved there.